packages/local-environment/src/actor.ts

```diff
@@ -24,17 +24,17 @@ export abstract class Actor {
 
 export abstract class DockerizedActor extends Actor {
   
-  docker: Docker;
-  container?: Docker.Container;
+  docker: Dockerode;
+  container?: Dockerode.Container;
   events: DockerEmitter;
   private eventsStream?: NodeJS.ReadableStream;
   logMatcher?: StreamMatcher;
 
   log = bunyan.createLogger({ name: "localenv" });
 
-  constructor(name: string, actorType: string) {
-    super(name, actorType)
-    this.docker = new Docker();
+  constructor(name: string, actorType: string, docker: Dockerode) {
+    super(name, actorType);
+    this.docker = docker;
```

Comment:
Probably a optional parameter. if not present fallback to `new Dockerode()`

Reply:
Addressed